bot/cogs/sudo.py

```diff
@@ -121,10 +125,8 @@ async def botstatus(self, ctx: Context, status: str, status_info: t.Literal["pla
                     status=Status.online
                 )
                 await ctx.send(f"Successfully changed playing status to **{status_info}**")
-            except InvalidArgument as e:
-                await ctx.send(e)
-            except Exception as e:
-                await ctx.send(e)
+            except Exception as error:
+                await ctx.send(error)
```

Comment:
Do we really need to handle every possible exception here? We should always try to avoid excepting everything.

There are multiple occurrences of this, please address all of them

Reply:
I'm just keeping it as it was. I'm not sure it's optimal, but that's how it was written

Reply:
Fixed those in sudo

Reply:
Though it's not the best possible error managing...